src/test/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/integration/IntegrationTestBase.kt

```diff
@@ -39,12 +43,14 @@ abstract class IntegrationTestBase : TestBase() {
       hmppsAuthMockServer.stubGrantToken()
 
       prisonApiMockServer.start()
+      offenderSearchMockServer.start()
     }
 
     @Suppress("unused")
     @AfterAll
     @JvmStatic
     fun stopMocks() {
+      offenderSearchMockServer
```

Comment:
Should this be `offenderSearchMockServer.stop()`?

Reply:
durrr